pulsar/internal/commands_test.go

```diff
@@ -37,3 +37,19 @@ func TestConvertStringMap(t *testing.T) {
 	assert.Equal(t, "1", m2["a"])
 	assert.Equal(t, "2", m2["b"])
 }
+
+func TestDecodeBatchPayload(t *testing.T) {
+	payload := []byte{0, 0, 0, 2, 24, 12, 104, 101, 108, 108, 111, 45, 112, 117, 108, 115, 97, 114} // hello-pulsar
```

Comment:
Seems this is batch payload, contains a single message of payload "hello-pulsar"